src/controllers/adminController.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+const Album = require('../models/Album');
+
+const adminHomePage = async (req, res) => {
+  const allAlbums = await Album.find({});
```

Comment:
Since the first task that we're trying to enable for the admin right now is editing prices for albums with a price of $0.00; maybe it'll be helpful to have those at the top of the table? So we could do an `order by` here to return the fetched albums in ascending order of price?

Reply:
This is good idea, thank you very much!

Reply:
<img width="400" alt="screen" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/95896159/f2a80eda-5930-4690-8a6c-e9f8d5fdaf11">